src/Header.tsx

```diff
@@ -7,6 +7,44 @@ import { AccountService } from './services/AccountService';
 import { HeaderDateRangeSelect } from './components/HeaderDateRangeSelect';
 import { ExternalLink } from './components/ExternalLink';
 import { AiOutlineGithub } from 'react-icons/ai';
+import styled, { createGlobalStyle } from 'styled-components';
+import { headerHeightPx } from './helpers/app-layout';
+
+// HACK There is no way to style Navbar.Collapse without using class names
+const navbarCollapseClassName = 'styled-navbar-collapse';
+const NavbarCollapseGlobalStyle = createGlobalStyle`
+  @media not screen and (min-width: 768px) {
+    .${navbarCollapseClassName} {
+      position: fixed;
+      top: ${headerHeightPx}px;
+      left: 0;
+      width: 100%;
+      max-height: calc(100vh - ${headerHeightPx}px);
+      overflow: hidden auto;
+      z-index: 30;
+      background: var(--light);
+      padding: 10px 30px;
+      border-bottom: 1px solid #dee2e6;
+
+      form {
+        margin: 5px 0;
+      }
+    }
+  }
+`;
+
+const LeftNavGroup = styled.div`
+  @media (min-width: 768px) {
+    margin-left: 1.5rem;
+    margin-right: auto;
+  }
+`;
+
+const GisaidText = styled.span`
```

Comment:
We can't change the way we show the GISAID note. We are obligated to write "Enabled by data from [GISAID logo]" by a written agreement.

Reply:
I changed it to never hide that text: 
![gisaid-full-text](https://user-images.githubusercontent.com/1489115/118151514-015ab700-b414-11eb-9eba-bc651457ac58.png)